sdcm/sct_provision/aws/cluster.py

```diff
@@ -0,0 +1,341 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2021 ScyllaDB
+
+import abc
+from functools import cached_property
+from typing import List, Dict, Optional
```

Comment:
Starting with python 3.9 we can use "list" and "dict" directly.

Reply:
I remember that there was some problem regarding that recently, not sure it is fixed